src/GitHub.Extensions/StringExtensions.cs

```diff
@@ -36,6 +36,16 @@ public static bool IsNullOrEmpty([AllowNull]this string s)
             return string.IsNullOrEmpty(s);
         }
 
+        public static bool IsNotNullOrEmptyOrWhiteSpace([AllowNull]this string s)
+        {
+            return !s.IsNullOrEmptyOrWhiteSpace();
+        }
+
+        public static bool IsNullOrEmptyOrWhiteSpace([AllowNull] this string s)
+        {
+            return string.IsNullOrEmpty(s) || string.IsNullOrWhiteSpace(s);
```

Comment:
Nice catch :smile: I missed that `IsNullOrWhiteSpace` returns true for null, empty or white space, I thought it was only true for null or white space...